src/PropertiesHandler.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Queries for all predicates of a path subject
+ *
+ */
+export default class PropertiesHandler {
```

Comment:
Maybe we should call it `PredicatesHandler` internally for clarity (but just expose it through `.properties`—or perhaps additionally as `.predicates`).